src/spootnik/reporter.clj

```diff
@@ -249,7 +249,14 @@
     (error e "captured exception")
     (when raven
       (try
-        (raven/capture! raven (:dsn sentry) e)
+        (raven/capture! {:http raven} (:dsn sentry) e)
+        (catch Exception e
+          (error e "could not send capture")))))
+  (capture! [this e tags]
+    (error e "captured exception")
+    (when raven
+      (try
+        (raven/capture! {:http raven} (:dsn sentry) e tags)
```

Comment:
is this code duplication avoidable? `(capture! [this e] (capture! [this e {}])` ?

Reply:
I would like to as well, but I'll do this when I can prove that it's indeed equivalent (after I have added tests).

For now I aimed at doing exactly like other cases (see for example inc! dec! and mark! in the same file).